src/details/ArborX_DetailsNode.hpp

```diff
@@ -46,7 +46,7 @@ struct Node
 KOKKOS_INLINE_FUNCTION constexpr Node
 makeLeafNode(std::size_t permutation_index, Box box) noexcept
 {
-  return {{-1, static_cast<int>(permutation_index)}, std::move(box)};
+  return {{-1, static_cast<int>(permutation_index)}, box};
```

Comment:
Not sure about this change.

Reply:
`Box` is trivially copyable so the move doesn't do anything here.

Reply:
But if `makeLeafNode` is called with direct initialization, wouldn't this create an extra  copy?